openstack/apigw/dedicated/v2/certificates/requests.go

```diff
@@ -0,0 +1,68 @@
+package certificates
+
+import "github.com/chnsz/golangsdk"
+
+type CertOpts struct {
+	// The certificate name.
+	Name string `json:"name" required:"true"`
+	// The certificate content.
+	Content string `json:"cert_content" required:"true"`
+	// The private key of the certificate.
+	PrivateKey string `json:"private_key" required:"true"`
+	// The certificate type. The valid values are as follows:
+	// + instance
+	// + global
+	Type string `json:"type,omitempty"`
+	// The dedicated instance ID to which the certificate belongs.
+	// If the certificate type is global, the instance ID is common by default.
```

Comment:
what's **common** means?

Reply:
the default value of the global certificates.
The APIs will not return empty and the value 'common' is support...